modules/nf-core/cafe/main.nf

```diff
@@ -0,0 +1,60 @@
+process CAFE {
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cafe:5.1.0--h43eeafb_0':
+        'biocontainers/cafe:5.1.0--h43eeafb_0' }"
+
+    input:
+    path(infile)
+    path(tree)
+
+    output:
+    path(".")     , emit: cafe
```

Comment:
Yes, I can add the meta.id back in (I just wasn't sure it was needed, but I think it is now). 
I added the whole output dir, as it may be that the user wanted all the output.
The output file names can change, depending on the model argument in the cafe script, 
<model>_counts.tab

So thats why I need to use :
path("*_count.tab") , emit: cafe_base_count

Ok, I will add in meta.id